main.nf

```diff
@@ -23,7 +23,18 @@ include { getGenomeAttribute      } from './subworkflows/local/utils_nfcore_nall
 
 /*
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-    NAMED WORKFLOW FOR PIPELINE
+    GENOME PARAMETER VALUES
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+*/
+
+// TODO nf-core: Remove this line if you don't need a FASTA file
+//   This is an example of how to use getGenomeAttribute() to fetch parameters
+//   from igenomes.config using `--genome`
+// params.fasta = getGenomeAttribute('fasta')
```

Comment:
It doesn't look like we are using the `getGenomeAttribute` function yet. Something for a future refactoring?

Reply:
Yes, that's the plan!